math/gunits/Portfile

```diff
@@ -1,10 +1,11 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
+PortGroup           compiler_blacklist_versions 1.0
 
 name                gunits
 version             2.24
-revision            0
+revision            1
```

Comment:
As documented at https://guide.macports.org, the revision should not have been increased because nothing has changed for anyone who was able to install the port before (but it cannot now be decreased).